### DISCUSSION TITLE
MP4 Pictures Shown Black on Zoom Video (Solution found)

### DISCUSSION THREAD

**Discussion by Tdanforth** [sircharlo/meeting-media-manager#146] [General]
I converted 2 JPG talk pictures to be used. When I opened the pictures using the video option on Zoom, it was a black screen. The picture did not show in the video player. During the talk, I opened jw-meeting-media-fetcher and converted the pictures again. When it came for the 2nd picture, it did the same thing. This is the first time this happened to me.

I almost thought there was an issue with the pictures, but then I saw there was a new update available. I updated to 5.8.6. I tested out the same MP4 pictures and they worked.

Not sure if the update fixed it or the Zoom restart. Just thought I would share this in case some had the same issue.

**Comment by sircharlo** [comment]
> I converted 2 JPG talk pictures to be used. When I opened the pictures using the video option on Zoom, it was a black screen. The picture did not show in the video player. During the talk, I opened jw-meeting-media-fetcher and converted the pictures again. When it came for the 2nd picture, it did the same thing. This is the first time this happened to me.
> 
> I almost thought there was an issue with the pictures, but then I saw there was a new update available. I updated to 5.8.6. I tested out the same MP4 pictures and they worked.
> 
> Not sure if the update fixed it or the Zoom restart. Just thought I would share this in case some had the same issue.

Absolutely. I've been corresponding with Zoom support regarding this same issue; they confirmed it was a problem in their MP4 rendering process and that they fixed it in the latest Zoom update.
